### PR TITLE
Certificate Error

### PR DESCRIPTION
**Pull request by RRudder**

### CODE REVIEW

File: submissions/description/server_security_misconfiguration/insecure_ssl/certificate_error/template.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+When a web page is loaded, the browser makes a series of checks against the website’s Secure Socket Layer (SSL) certificate. When these checks fail, the browser blocks the webpage from loading and issues a certificate error. An attacker can leverage this information to identify what caused the certificate error and execute a Person-in-The-Middle (PitM) attack to view and modify information that passes between the vulnerable endpoints.
+
+## Business Impact
+
+Certificate errors can lead to reputational damage for the business due to a loss in confidence and trust by users who attempt to connect to the webpage and receive a certificate error.
+
+## Steps to Reproduce
+
+1. Navigate to the following URL to find a certificate error is thrown by the browser: {{URL}}
+1. Sensitive information can be discovered by {{action}}
```
Comment (evildaemond) [inline submissions/description/server_security_misconfiguration/insecure_ssl/certificate_error/template.md:14]: What is sensitive about a certificate error?

**Review by evildaemond — CHANGES_REQUESTED** [review]
I think this needs some rephrasing then it'll be okay

File: submissions/description/server_security_misconfiguration/insecure_ssl/certificate_error/template.md (hunk 1 above)
Comment (RRudder) [reply submissions/description/server_security_misconfiguration/insecure_ssl/certificate_error/template.md:14]: I'm not sure. It was in the previous template's steps to reproduce so I left it in there to help researchers convey impact.  I wasn't sure if I was missing something with certificate errors where an attacker could discover sensitive info. 

I can remove this step if there's nothing aside from viewing the certificate error needed.

Comment (evildaemond) [reply submissions/description/server_security_misconfiguration/insecure_ssl/certificate_error/template.md:14]: Likely that's the best option here, just to remove it

**Review by evildaemond — APPROVED** [review]
LGTM

**Pull request merged**